frontend/src/components/TableUnratedTaskRow.tsx

```diff
@@ -227,12 +227,18 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
           </StylesProvider>
         </div>
       </td>
-      <td className="styledTd nowrap">
-        <TaskRatingsText task={task} />
-      </td>
-      <td className="styledTd">{new Date(createdAt).toLocaleDateString()}</td>
-      <td className="styledTd textAlignEnd nowrap" style={{ width: '202px' }}>
-        {userInfo && isUnrated(userInfo)(task) && (
+      <td className={classes(css.buttonContainer)}>
+        {type === RoleType.Admin && (
+          <button
+            style={{ marginRight: '10px' }}
+            className={classes(css['button-small-outlined'])}
+            type="button"
+            onClick={notifyUsersClicked}
+          >
+            <Trans i18nKey="Notify" />
+          </button>
+        )}
+        {taskAwaitsRatings(task, userInfo) ? (
           <a
             href={modalLink(ModalTypes.TASK_RATINGS_INFO_MODAL, {
               taskId: task.id,
```

Comment:
I am not sure if this link is needed (the correct url is shown without it too), but shouldn't the ModalTypes be RATE_TASK_MODAL?